### PR TITLE
Fixed create app entity intersections in VR, increased default voxel resolution

### PR DESCRIPTION
**Pull request by ksuprynowicz**

### CODE REVIEW

**Comment by ksuprynowicz** [comment]
This fixes annoying bug with unexpectedly selecting entities in VR when moving entity with handle that is obsucred by another object and bug with voxeld being edited when clicking on tablet. Defalut voxel resolution is also increased to higher value.

**Comment by ksuprynowicz** [comment]
Anyone would like to review this? It fixes bugs that made create app almost unusable in VR in earlier versions so I would like to include it in the upcoming release.

**Comment by ksuprynowicz** [comment]
I tested it on Windows in VR and everything seems to work well. It didn't break anything in Desktop mode on Linux either.

File: scripts/system/controllers/controllerModules/inEditMode.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ Script.include("/~/system/libraries/utils.js");
                         }));
                     } else {
                         if (this.selectedTarget.type === Picks.INTERSECTED_ENTITY) {
+                            print("selectEntity message")
```
Comment (daleglass) [inline scripts/system/controllers/controllerModules/inEditMode.js:83]: Probably could use the wantDebug check

File: scripts/system/controllers/controllerModules/inEditMode.js [hunk 2]
```diff
@@ -88,6 +89,7 @@ Script.include("/~/system/libraries/utils.js");
                                 intersection: this.selectedTarget.intersection
                             }));
                         } else if (this.selectedTarget.type === Picks.INTERSECTED_OVERLAY) {
+                            print("selectOverlay message")
```
Comment (daleglass) [inline scripts/system/controllers/controllerModules/inEditMode.js:92]: here too

**Review by daleglass — CHANGES_REQUESTED** [review]

**Comment by AnimatedMannequin** [comment]
Here are some builds you may use for testing: 

**macOS-10.15, full**
 - https://public.overte.org/build/overte/PR191/Overte-PR191-d572fb3.dmg

**windows-2019, full**
 - https://public.overte.org/build/overte/PR191/Overte-PR191-d572fb3.exe

**Pull request merged**